pkg/reconciler/testing/pstest.go

```diff
@@ -52,10 +52,10 @@ func SubscriptionWithTopic(id string, tid string) PubsubAction {
 	}
 }
 
-func TopicAndSub(id string) PubsubAction {
+func TopicAndSub(tid, sid string) PubsubAction {
```

Comment:
Why did this change? Is there a need to use different topic and subscription names?

Reply:
The new PubSubReconciler tests need this. And I think as a helper method, this should allow different names anyway.